include/wx/textctrl.h

```diff
@@ -805,7 +807,10 @@ class WXDLLIMPEXP_CORE wxTextCtrlBase : public wxControl,
     }
     virtual void SetValue(const wxString& value) override
     {
+       auto old = m_maxLengthAllowed;
+       m_maxLengthAllowed = false;
```

Comment:
This GTK-specific hack should be in GTK-specific code if it's needed at all, not here.